packages/colorpicker/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@zendeskgarden/react-colorpicker",
+  "description": "Components related to colorpicker in the Garden Design System",
+  "license": "Apache-2.0",
+  "author": "Zendesk Garden <garden@zendesk.com>",
+  "homepage": "https://garden.zendesk.com/",
+  "repository": "https://github.com/zendeskgarden/react-components",
+  "bugs": {
+    "url": "https://github.com/zendeskgarden/react-components/issues"
+  },
+  "version": "8.29.3",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "../../utils/scripts/build.sh",
+    "build:demo": "../../utils/scripts/build-demo.sh",
+    "start": "../../utils/scripts/start.sh"
+  },
+  "sideEffects": false,
+  "types": "dist/typings/index.d.ts",
+  "dependencies": {
+    "@zendeskgarden/container-utilities": "^0.5.5",
+    "@zendeskgarden/react-forms": "^8.29.3",
+    "@zendeskgarden/react-modals": "^8.29.3",
+    "polished": "^4.0.0",
+    "react-merge-refs": "^1.1.0"
+  },
+  "peerDependencies": {
+    "@zendeskgarden/react-theming": "^8.1.0",
+    "prop-types": "^15.6.1",
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0",
+    "styled-components": "^4.2.0 || ^5.0.0"
+  },
+  "devDependencies": {
+    "@types/lodash.throttle": "4.1.6",
+    "@zendeskgarden/react-theming": "^8.0.0",
+    "@zendeskgarden/svg-icons": "6.27.0",
+    "lodash.throttle": "4.1.1"
```

Comment:
If this is used within any non-storybook code we need to include it as a `dependency` to ensure that it's included for consumers.

Reply:
Moved to deps. Thanks!